### PR TITLE
TC_00.004.01 | Verify a User can create Folder from sidebar menu using New Item menu link, with unique name and default configuration

### PR DESCRIPTION
**Pull request by LanaUno**
[TC_00.004.01](https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/164)

### CODE REVIEW

**Review by Irina-Didorenko — APPROVED** [review]

**Comment by adiuzhikov** [comment]
Kindly adhere to the project guidelines during the test creation process:

1. Branch name: Your initials / TC number (e.g., sa/TC_00.004.01).
2. PR title: TC title (e.g., TC_00.004.01 | Verify a User can create a Folder from the sidebar menu using the New Item menu link, with a unique name and default configuration).
3. PR description: Link to your TC card on board.

File: cypress/e2e/createFolder.cy.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    cy.get("#jenkins-name-icon").click();
+    cy.get("span")
+      .contains("New Folder")
+      .trigger("mouseover")
+      .should("be.visible");
+    cy.get(".jenkins-table__link > .jenkins-menu-dropdown-chevron").click({
+      force: true,
+    });
+    cy.get("button.jenkins-dropdown__item").click();
+    cy.get(".jenkins-button--primary").click();
```
Comment (adiuzhikov) [inline cypress/e2e/createFolder.cy.js:20]: Lines 16-20 of the test are unnecessary, as the .cleanData() method is executed prior to each test.

Comment (LanaUno) [reply cypress/e2e/createFolder.cy.js:20]: > Lines 16-20 of the test are unnecessary, as the .cleanData() method is executed prior to each test.

I did the first test before I pulled the project, and it crashed, so these lines were needed. What would be your advice - to delete them or leave them?

**Comment by LanaUno** [comment]
> Kindly adhere to the project guidelines during the test creation process:
> 
> 1. Branch name: Your initials / TC number (e.g., sa/TC_00.004.01).
> 2. PR title: TC title (e.g., TC_00.004.01 | Verify a User can create a Folder from the sidebar menu using the New Item menu link, with a unique name and default configuration).
> 3. PR description: Link to your TC card on board.
> 4. Specification File Name: US Name (e.g., newItemCreateFolder.cy.js)

_Thank you, I will_

File: cypress/e2e/createFolder.cy.js (hunk 1 above)
Comment (adiuzhikov) [reply cypress/e2e/createFolder.cy.js:20]: > _I did the first test before I pulled the project, and it crashed, so these lines were needed. What would be your advice - to delete them or leave them?_

Feel free to leave them for now. Once you pull the code from main, you can refactor your test and make any additional changes you’d like. Don't forget to rename your spec file!

**Review by adiuzhikov — APPROVED** [review]

File: cypress/e2e/createFolder.cy.js (hunk 1 above)
Comment (LanaUno) [reply cypress/e2e/createFolder.cy.js:20]: _Thank you for feedback_

**Pull request merged**